components/project-list/project-list.jsx

```diff
@@ -66,15 +88,6 @@ export default React.createClass({
       return entries.sort((a,b) => {
         return bookmarkedIdArray.indexOf(a.id.toString()) > bookmarkedIdArray.indexOf(b.id.toString());
```

Comment:
If that's the case don't worry about it. We just need to do the sorting once - for the initial /fav (bookmark) page render. :bowtie: